synapse/push/__init__.py

```diff
@@ -116,6 +116,7 @@ class PusherConfig:
     last_stream_ordering: int
     last_success: Optional[int]
     failing_since: Optional[int]
+    enabled: bool
```

Comment:
This looks correct due to the COALESCEs in the relevant queries. But I think you said elsewhere that this field is meant to be nullable when given to clients. (MSC: "A new nullable field enabled is added to the Pusher model.")

Does "nullable" in that sentence actually mean "optional"? In other words, does it mean that 
- the field may not be present,
- but if it is, it must be a boolean?

Reply:
The MSC does say nullable but as you pointed out it's likely a mistake. The field that's nullable is `device_id` in `GET /pushers` responses, which is implemented in #13831